packages/template-retail-react-app/app/pages/checkout/util/checkout-context.js

```diff
@@ -33,17 +33,16 @@ export const CheckoutProvider = ({children}) => {
         sectionError: undefined
     })
 
-    const CheckoutSteps = {
-        Contact_Info: 0,
-        Shipping_Address: 1,
-        Shipping_Options: 2,
-        Payment: 3,
-        Review_Order: 4
-    }
+    const CheckoutStepsList = [
+        'Contact_Info',
+        'Shipping_Address',
+        'Shipping_Options',
+        'Payment',
+        'Review_Order'
+    ]
+    const CheckoutSteps = CheckoutStepsList.reduce((acc, step, idx) => ({...acc, [step]: idx}), {})
 
-    const getCheckoutStepName = (step) => {
-        return Object.keys(CheckoutSteps).find((key) => CheckoutSteps[key] === step)
-    }
+    const getCheckoutStepName = (step) => CheckoutStepsList[step]
```

Comment:
This is technically bonus content, since it was pretty trivial to change `getCheckoutStepName` from O(n) to O(1).